Tests/Pcap++Test/main.cpp

```diff
@@ -229,6 +229,7 @@ int main(int argc, char* argv[])
 	PTF_RUN_TEST(TestPcapLiveDeviceNoNetworking, "no_network;live_device");
 	PTF_RUN_TEST(TestPcapLiveDeviceStatsMode, "live_device");
 	PTF_RUN_TEST(TestPcapLiveDeviceBlockingMode, "live_device");
+	PTF_RUN_TEST(TestPcapLiveDeviceBlockingModePollTimeout, "live_device");
```

Comment:
You also need to add `TestPcapLiveDeviceBlockingModeNotTimeoutWithoutPoll` here if you want to run it...